tracing/spec/src/main/asciidoc/microprofile-telemetry-tracing-spec.asciidoc

```diff
@@ -405,7 +407,7 @@ Behavior when customizer changes other properties than those listed in the spec
 
 [[sec:semantic-conventions]]
 === Semantic Conventions
-The https://github.com/open-telemetry/opentelemetry-specification/tree/v{otel-spec-version}/specification/trace/semantic_conventions[Trace Semantic Conventions] for Spans and Attributes must be followed by any compatible implementation.
+The https://github.com/open-telemetry/opentelemetry-java/tree/release/v{otel-java-version}/semconv/src/main/java/io/opentelemetry/semconv[Trace Semantic Conventions] for Spans and Attributes must be followed by any compatible implementation.
```

Comment:
```suggestion
The https://github.com/open-telemetry/opentelemetry-java/tree/v{otel-java-version}/semconv/src/main/java/io/opentelemetry/semconv[Trace Semantic Conventions] for Spans and Attributes must be followed by any compatible implementation.
```

URL is wrong so I've corrected it, but in addition, these files seem to have less information in them and less detail than the [main OTel spec](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.20.0/specification/trace/semantic_conventions).

In particular, it doesn't say which attributes are recommended, optional and required.